pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[tool.poetry]
+name = "clairvoyance"
+version = "2.0.0"
+description = "Obtain GraphQL API Schema even if the introspection is not enabled"
+authors = [
+    "Nikita Stupin <nikitastupin.com>",
```

Comment:
Would you be willing to include your name too?

Reply:
Or is there the `contributors` field?

Reply:
Yeah I defintely think you deserve it @c3b5aw ;)

Reply:
> Or is there the `contributors` field?

https://python-poetry.org/docs/pyproject/
No there is only "authors" and "maintainers". 

Concerning the "author" field, please know that we started working on improving Clairvoyance about 8 months ago with @MdotTIM (we all work for https://escape.tech). However we never reached a sufficient state of quality to give you back those changes ;)
@c3b5aw restarted completely from your repo and reached this amazing results during his Open Source Days :) I absolutely think he deserves being co-author with you on this v2.

At the moment you are officially the only maintainer of the repo but we love contributing to the GraphQL Ecosystem so we could help maintaining the repo if this is something you're okay with ;) 

Best,

Antoine
Co-founder & CTO @Escape.tech